### DISCUSSION TITLE
warn if simultaneous query started

### DISCUSSION THREAD

**Discussion by rdp** [dbeaver/dbeaver#13426] [Q&A] [answered]
It confused me when I ran two long queries in two tabs, to the same database.
It says it's running but then it appears that one waits for the other to finish first?  With no hint that that's the case.

**Describe the solution you'd like**
Ideally both queries would run simultaneously.  If not then at least a different state like "waiting for other query to finish" would be nice to display in the UI.

**Describe alternatives you've considered**
See above.
Thank you.

**Comment by p91paul** [comment, marked as answer]
somewhere in preferences there's an option called something like "open a different connection for each editor", try to enable it!

**Comment by rdp** [comment]
Yes that does work ("Database -> Editors -> SQL Editors"), thanks!
It's surprising to me that this wouldn't be the default...
As it is today it's just confusing "why does it say fetching table data *forever* when I run this SQL query..." so it would be nice to either make it the default or have some other status shown ("waiting for connection that's being used in other tab...") to reduce the confusion...
